pages/people.tsx

```diff
@@ -31,13 +37,14 @@ function PeoplePage() {
 
       <Table
         data={{
-          head: ["ID", "Full Name"],
+          head: ["Image", "Full Name"],
           body: data?.people.map((person: PersonShape) => [
-            person.id,
+            <Image src={person.image} className="max-w-16" />,
```

Comment:
totally arbitrary width&mdash;really just enough to keep the table rows from taking up half the viewport